src/rata-summary-workspace/rata-summary-workspace.service.ts

```diff
@@ -72,7 +73,9 @@ export class RataSummaryWorkspaceService {
     // Checks if RATA Test is valid.
     await this.rataService.getRataById(rataId);
```

Comment:
This uses the newly created RATA, so the transaction should be passed as a parameter (and the method should be updated to use it):
```suggestion
    await this.rataService.getRataById(rataId, trx);
```

Reply:
updated.